integration/spark-common-test/src/test/scala/org/apache/carbondata/integration/spark/testsuite/timeseries/TestTimeSeriesCreateTable.scala

```diff
@@ -319,6 +326,53 @@ class TestTimeSeriesCreateTable extends QueryTest with BeforeAndAfterAll {
     assert(e.getMessage.equals(s"$timeSeries should define time granularity"))
   }
 
+  test("test timeseries create table 19: should support if not exists") {
+    sql("DROP DATAMAP IF EXISTS agg1 ON TABLE mainTable")
+    try {
```

Comment:
no need for try block. If any exception if thrown the test case will fail

Reply:
ok, done